hoodie-client/src/main/java/com/uber/hoodie/io/HoodieAppendHandle.java

```diff
@@ -158,6 +157,7 @@ public void doAppend() {
 
         List<IndexedRecord> recordList = new ArrayList<>();
         List<String> keysToDelete = new ArrayList<>();
+        List<HoodieLogBlockMetadata> metadata = new ArrayList<>();
```

Comment:
style: This can be 1 line

`List<HoodieLogBlockMetadata>  metadata = Arrays.asList(new HoodieInstantTime(commitTime));`